indexstyle.css

```diff
@@ -79,26 +79,41 @@ ul {
 	display: flex;
 	justify-content: center;
 	padding: 0;
+	margin-bottom: 5px;
 }
 
 li {
 	list-style-type: none;
 }
 
 li a {
-	padding: 12px;
+	padding: 12px 20px;
 	text-decoration: none;
 	color: #000500;
-	border-right: 1px solid #000500;
+	border-right: 1px solid #A9A9A9;
 	margin: 0;
 }
 
 .lastnav a {
 	border-right: none;
 }
 
+.thirdnav a {
+	border-left: 1px solid #A9A9A9;
+}
+
 li a:hover {
-	font-weight: 400;
+	text-shadow: 0 0 0.4px #000500;
+	transition-delay: 0.08s;
```

Comment:
Do you want the effect to fade away as well or just instantly disapear?

Reply:
ok with instant disappearance